cat/js_css/cat_keys.js

```diff
@@ -7,9 +7,13 @@
 // Key Bindings _____________________________________________
 
 document.onkeydown = function() {  //   KEY DOWN
+  let keyDirections = {'a': 'left', 'd': 'right'}

```

Comment:
Keep filling this thing out!

Reply:
Excellent suggestions! Thanks for the contribution!